hack/run-external-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+if [ "$FEATURES" == "" ]; then
+	echo "[ERROR]: No FEATURES provided"
+	exit 1
+fi
+
+export failed=false
+
+for feature in $FEATURES; do
+  
+  test_entry_point=external-tests/${feature}/test.sh
+  if [[ ! -f $test_entry_point ]]; then
+    echo "[INFO] Feature '$feature' does not have external tests entry point"
+    continue
+  fi
+  echo "[INFO] Running external tests for $feature"
+  set +e
+  if ! $test_entry_point; then
+    echo "[WARN] External tests failed for $feature"
```

Comment:
could we capture a summary of all the tests suites that failed and display it as the last messages? maybe use a bash array and echo out all the tests that failed so it's obvious where to look in the output at first glance?

Reply:
Makes sense, will do.

Reply:
Done